utilities/metadata_parser.py

```diff
@@ -59,12 +60,12 @@ def populate_from_metadata(metadata_filename=None):
 
             if missing_metadata:
                 print(f'WARNING: Line {line_id+1} is incomplete (skipping).\
-                        \n\tMissing fields: {missing_metadata} - FIXME in the metadata!')
+                        \n\tMissing fields: {missing_metadata} - FIX ME in the metadata!')
```

Comment:
Just FYI: 'FIXME' (no spaces) is typical in a codebase so that you can search for it project-wide. (Here it doesn't matter because it's just in a message to the user...)